Add russian translation

Katrovsky opened the pull request.

Katrovsky committed:
Add russian locale

YorVeX commented:
Thanks a lot for submitting a new translation!

Please note that double quotes (") are used to indicate the start and end of a text string, meaning if you use it within the text, the text will prematurely end there.

So for example this will never show the "BeamSender" part, it will just stop after используется:
`IdentifierText="Идентификатор отправителя. Это используется для идентификации отправителя на приемнике. По умолчанию, если пусто, используется "BeamSender"."`

To use double quotes within the text, they need to be escaped like this: `\"`

Meaning the above example needs to look like this:
`IdentifierText="Идентификатор отправителя. Это используется для идентификации отправителя на приемнике. По умолчанию, если пусто, используется \"BeamSender\"."`

Would you please be so kind to fix this for the PR?

What could help finding all the cases where it occurs is the syntax highlighting if you check this PR, the affected items will have a different text color:
![image](https://github.com/user-attachments/assets/ec19e6ff-f71f-4e01-88d7-8b5a3d2fff64)

![image](https://github.com/user-attachments/assets/5dd72a69-6446-4409-9878-f007cf43d7de)

Katrovsky commented:
Give me some time, please, I'll fix it. :)

Katrovsky committed:
Update ru-RU.ini

Carefully rechecked the entire text, corrected all errors. Thanks for waiting. :)

Katrovsky commented:
I probably should have written about it explicitly, so I'm sorry: everything is fixed.
